x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -142,6 +150,90 @@ export async function getPackages(
   return packageListWithoutStatus;
 }
 
+export async function getInstalledPackages(options: {
+  savedObjectsClient: SavedObjectsClientContract;
+  type?: PackageDataStreamTypes;
+  nameQuery?: string;
+  pageAfter?: SortResults;
+  pageSize: number;
```

Comment:
we have used `perPage` in other Fleet APIs for a similar param: https://github.com/elastic/kibana/blob/069550d72a5549b0fe45f3290c0d856ea2bfd309/x-pack/plugins/fleet/server/services/agents/crud.ts#L397